waiter/src/waiter/util/utils.clj

```diff
@@ -28,7 +28,8 @@
             [taoensso.nippy.compression :as compression]
             [waiter.status-codes :refer :all]
             [waiter.util.date-utils :as du]
-            [waiter.util.http-utils :as hu])
+            [waiter.util.http-utils :as hu]
+            [full.async :as fa])
```

Comment:
lexicographic order :(

Reply:
Is there a good plugin that does this?

Reply:
I used to use: https://plugins.jetbrains.com/plugin/12731-amethyst
But it is not compatible with my IntelliJ anymore :(